src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
the semantic reason for an id and class usage in an app is to have a line
between repeating objects and unique ones. Actually I would love to delete
all the css and write my own but that would not be backwards compatible.
due to prevent clashes and having discrete semantic elements I had to use
id, so each page had its own unique discrete sections too.

11 Haziran 2015 Perşembe tarihinde, Jonas <notifications@github.com
<javascript:_e(%7B%7D,'cvml','notifications@github.com');>> yazdı:

> In src/Template/Bake/Template/index.ctp
> <https://github.com/cakephp/bake/pull/104#discussion_r32177515>:
>
> > @@ -39,20 +39,21 @@ $fields = collection($fields)
> >      endforeach;
> >  %>
> >      </ul>
> > -</div>
> > -<div class="<%= $pluralVar %> index large-10 medium-9 columns">
> > +</nav>
> > +<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
>
> Same as above.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cakephp/bake/pull/104/files#r32177515>.
>